cylc/flow/etc/syntax/cylc.lang

```diff
@@ -90,7 +90,6 @@
         <keyword>xtriggers</keyword>
         <keyword>work sub-directory</keyword>
         <keyword>warning handler</keyword>
-        <keyword>visualization</keyword>
```

Comment:
I think that there are a number of other keywords that should be removed. Try running:
`./etc/bin/syntax_generator.py`.

Reply:
You're right; last commit cleans this up.